File: C/lista3 - vetores/exercicio_9.c
#include <conio.h>
#include <stdio.h>

int main() {
	int ages[3], i, totalHighStudents = 0;
	float averageHeights = 0, heights[3], sumHeights = 0;
	
	for (i= 0; i< 3; i++) {
		printf("Informe uma altura: ");
		scanf("%f", &heights[i]);
		printf("Informe uma idade: ");
		scanf("%d", &ages[i]);
		
		sumHeights += heights[i];
	}
	
	averageHeights = sumHeights / 3;
	
	for (i= 0; i< 3; i++) {
		if (ages[i] >= 13 && heights[i] >= averageHeights) {
			totalHighStudents++;
		}
	}
	
	printf("Existem %d alunos com mais de 13 anos acima da media de altura", totalHighStudents);
	
	getch();
	return 0;
}

File: Python/lista3 - listas/exercicio_9.py
students = []
total_students = 30
student = []

for i in range(0, total_students):
    student.append(int(input('Informe a idade do aluno: ')))
    student.append(float(input('Informe a altura do aluno: ')))
    students.append(student)

sum_height = 0.0

for student in students:
    sum_height += student[1]

average_height = sum_height / float(total_students)

total_high_students = 0
for student in students:
    if (student[0] >= 13) and (student[1] >= average_height):
        total_high_students += 1

print('Existem %d alunos com mais de 13 anos acima da media de altura' % total_high_students)
